bin/proxy.js

```diff
@@ -32,7 +34,7 @@ const array = (value) => {
 function unsecureCookie(req, res, proxyRes) {
   const proxyCookies = array(proxyRes.headers['set-cookie']);
   // eslint-disable-next-line no-param-reassign
-  proxyRes.headers['set-cookie'] = proxyCookies.map((rewrittenCookie) => {
+  proxyRes.headers['set-cookie'] = proxyCookies.map(rewrittenCookie => {
```

Comment:
ESLint used to yell at me when I was doing this (which is actually valid..)

Reply:
I *thoroughly* hate this syntax. I think we should ban it.